### PR TITLE
Node egress traffic shaping

### PR DESCRIPTION
**Pull request by yogananth-subramanian**
### Description

Patch adds a scenario to create variations in egress traffic of a Node's interface using the tc and Netem.

### CODE REVIEW

**Comment by yogananth-subramanian** [comment]
rerun

File: docs/network-chaos.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+The following environment variables can be set on the host running the container to tweak the scenario/faults being injected:
+
+ex.)
+`export <parameter_name>=<value>`
+
+Parameter               | Description                                                           | Default
+----------------------- | -----------------------------------------------------------------     | ------------------------------------ |
+DURATION                | Duration in seconds - during with network chaos will be applied.         | 300                                  |
+NODE_NAME               | Node name to inject faults in case of targeting a specific node; Can set multiple node names separated by a comma      | ""                                   |
```
Comment (chaitanyaenr) [inline docs/network-chaos.md:31]: We might want to mention that a random node is picked matching the label if NODE_NAME is not set. Thoughts?

**Review by chaitanyaenr — APPROVED** [review]
Minor nit in the docs, other than that LGTM.

File: docs/network-chaos.md (hunk 1 above)
Comment (yogananth-subramanian) [reply docs/network-chaos.md:31]: sounds good, will add it to LABEL_SELECTOR

**Review by paigerube14 — APPROVED** [review]
LGTM

**Pull request merged**